release_automation.sh

```diff
@@ -287,15 +288,15 @@ execute "git" "push" "-u" "origin" "HEAD"
 ohai "Create release branch in WordPress-Android"
 execute "git" "switch" "-c" "$WP_APPS_INTEGRATION_BRANCH"
 
-ohai "Update gutenberg-mobile ref"
-cd libs/gutenberg-mobile
-execute "git" "fetch" "--recurse-submodules=no" "origin" "$GB_MOBILE_PR_REF"
-execute "git" "checkout" "$GB_MOBILE_PR_REF"
-execute "git" "submodule" "update"
-cd ../..
+# Get the last part of the path from GB_MOBILE_PR_URL
+PULL_ID=${GB_MOBILE_PR_URL##*/}
 
-execute "git" "add" "libs/gutenberg-mobile"
-execute "git" "commit" "-m" "Release script: Update gutenberg-mobile ref"
+ohai "Update build.gradle file with the latest version"
+test -f "build.gradle" || abort "Error: Could not find build.gradle"
+sed -i'.orig' -E "s/ext.gutenbergMobileVersion = '(.*)'/ext.gutenbergMobileVersion = '${PULL_ID}-$GB_MOBILE_PR_REF'/" build.gradle || abort "Error: Failed updating gutenbergMobileVersion in build.gradle"
+
+execute "git" "add" "build.gradle"
+execute "git" "commit" "-m" "Release script: Update build.gradle gutenbergMobileVersion to ref"
 
 ohai "Update strings"
 execute "python" "tools/merge_strings_xml.py"
```

Comment:
It would appear this script was [recently modified](https://github.com/wordpress-mobile/WordPress-Android/commit/03002428012eb459b774927b70448d26d9ae0a43) and now requires [additional arguments](https://github.com/wordpress-mobile/WordPress-Android/blob/a89637138ba84c5576c1d7fb1a8c5c41d244c76a/tools/merge_strings_xml.py#L107-L109). So, we may need to update this command, although I'm not sure what parameters should be passed. However, we may be able to remove this script altogether, if I follow the discussion in p1618416871372200/1618248527.360700-slack-C0126N6JH7Z correctly. WDYT?

Reply:
> we may be able to remove this script altogether

I think this is the right approach because the WPAndroid release process is handling the string updates now.